wgsl/index.bs

```diff
@@ -9811,8 +9820,9 @@ For each [=formal parameter=] of a function, one or two tags are computed:
   <thead>
     <tr><th>Parameter Return Tag<th>Description
   </thead>
-  <tr><td><dfn noexport>ParameterReturnRequiredToBeUniform</dfn>
-      <td>The parameter [=shader-creation error|must=] be a [=uniform value=] in order for the [=return value=] to be a uniform value.
+  <tr><td><dfn noexport>ParameterReturnContentsRequiredToBeUniform</dfn>
+      <td>The parameter [=shader-creation error|must=] be a [=uniform value=] in order for the [=return value=] contents to be a uniform value.
```

Comment:
Nit: I think we should drop the "contents" here from "return value contents", since the return value is not a pointer.